app/index.html

```diff
@@ -3,8 +3,10 @@
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <base href="<%= APP_BASE %>">
   <title>My Angular 2 App</title>
+  <base href="<%= APP_BASE %>">
+  <!-- ignore: Aplha 32 router fix -->
+  <script>baseElement = document.querySelector('base');baseElement.attr = baseElement.getAttribute;</script>
```

Comment:
This is also what @gdi2290 did as well:
https://github.com/angular-class/angular2-webpack-starter/blob/0122d866cfcffb76150aa3047e6f9adb1959733d/src/public/index.html#L15-16